packages/web3/test/e2e/estimate_gas.test.ts

```diff
@@ -27,13 +27,13 @@ describe(`${getSystemTestBackend()} tests - estimateGas`, () => {
 	const provider = getSystemE2ETestProvider();
 	const blockData = getSystemTestBackend() === 'sepolia' ? sepoliaBlockData : mainnetBlockData;
 	const simpleEthTransaction: Transaction = {
-		from: getE2ETestAccountAddress(),
+		from: '0x0000000000000000000000000000000000000000',
```

Comment:
The function `getE2ETestAccountAddress()` was supposed to return `0x0000000000000000000000000000000000000000`
What is the reason for changing the values inside the constant `TEST_ACCOUNT_ADDRESS` and using the 0x0 address here?
Thanks,

Reply:
The test functions `getStorage` and `call` [fail ](https://github.com/web3/web3.js/actions/runs/7479805308/job/20357902163) on sepolia and [mainnet ](https://github.com/web3/web3.js/actions/runs/7479805308/job/20357902399#logs) because they expect `getE2ETestAccountAddress` to be a different address than 0x0000000000000000000000000000000000000000